package-lock.json

```diff
@@ -21,7 +21,6 @@
         "@types/react-dropzone": "^5.1.0",
         "@yudiel/react-qr-scanner": "^2.0.8",
         "mixpanel-browser": "^2.54.0",
-        "papaparse": "^5.5.2",
```

Comment:
is this intentional?

Reply:
This happened when I did an `npm install`. Not sure why.